app/src/main/java/br/edu/ifal/fiscalizaapp/screens/example/ExampleActivity.kt

```diff
@@ -4,134 +4,220 @@ import android.net.Uri
+                            variant = ButtonVariant.Link(),
+                            rightIcon = Icons.Default.ChevronRight,
+                            modifier = Modifier.fillMaxWidth()
+                        )
+
+                        Spacer(modifier = Modifier.height(32.dp))
+                        TextArea(
+                            value = description,
+                            onValueChange = { description = it },
+                            label = "Descrição",
+                            maxLength = 500
+                        )
+
+                        Spacer(modifier = Modifier.height(24.dp))
+                        Text(
+                            text = "3. FAQ - Accordion",
+                            style = MaterialTheme.typography.titleMedium
+                        )
+
+                        Spacer(Modifier.height(8.dp))
+
+                        CardBase {
+                            Column(
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .padding(12.dp),
+                                verticalArrangement = Arrangement.spacedBy(12.dp)
+                            ) {
+                                faqs.forEachIndexed { index, (question, answer) ->
+                                    Accordion(
+                                        question = question,
+                                        answer = answer,
+                                        expanded = expanded[index],
+                                        onToggle = { expanded[index] = !expanded[index] },
+                                        modifier = Modifier.fillMaxWidth()
+                                    )
+                                }
+                            }
+                        }
+
+                        Spacer(Modifier.height(16.dp))
+                        Row(
+                            modifier = Modifier.fillMaxWidth(),
+                            horizontalArrangement = Arrangement.spacedBy(12.dp)
+                        ) {
+                            OutlinedButton(
+                                onClick = {
+                                    uris = emptyList()
+                                    description = ""
+                                    for (i in expanded.indices) expanded[i] = false
+                                },
+                                modifier = Modifier.weight(1f)
+                            ) {
+                                Text("Limpar")
+                            }
+                            M3Button(
+                                onClick = {
+                                    // TODO: ação real (salvar/enviar)
+                                },
+                                modifier = Modifier.weight(1f)
+                            ) {
+                                Text("Enviar")
+                            }
```

Comment:
```suggestion
 
```

Não precisa desses botões, o FAQ não vai ser um componente com actions, apenas vai mostrar perguntas e respostas.

Além disso, quando for usar botões, utilize os botões padrão que @ludmilabss e eu fizemos